scripts/_utils/utils.py

```diff
@@ -152,6 +167,8 @@ def verify_image_integrity(file_url: str, mime: str, local: bool) -> Tuple[bool,
 
     if mime == 'image/svg+xml':
         return process_svg(file_url)
+    elif mime == 'image/png':
+        return remove_transparency(im, file_url)
     elif mime == 'image/gif':
```

Comment:
Don't we also want svg and gifs to have transparency removed?  Seems like they should both go through the png process AFTER they are svg or gif processed.

Reply:
yes; however, I was going to separate it into two different PRs as it's two different issues, would you like me to combine them?